assets/state-tray.desktop

```diff
@@ -0,0 +1,14 @@
+[Desktop Entry]
+Name=ActiveState Desktop
+GenericName=Language Runtime Manager
+Type=Application
+Comment=Manage ActiveState Platform Projects
+Exec="state-tray"
```

Comment:
Please use the full path. We don't want to have to rely on PATH for this, especially considering we'll be wanting to test multiple branches side by side.